File: tasks/oj/boj/c/1769.c
#include <stdio.h>
#include <string.h>

int main() {
  int n = 0, counts = 0;
  char gets[1000000] = "";
  scanf("%s", gets);
  for (int i = 0; i < strlen(gets); i++) {
    n += (int)gets[i] - 48;
  }
  if (strlen(gets) >= 2) {
    counts++;
  }
  while (n >= 10) {
    int new = 0;
    while (n != 0) {
      new += n % 10;
      n /= 10;
    }
    n = new;
    counts++;
  }
  printf("%d\n", counts);
  printf("%s\n", n == 3 || n == 6 || n == 9 ? "YES" : "NO");
}

File: tasks/online_judge/baekjoon/python/1769.py
gets = input()
n = 0
counts = 0
for s in gets:
    n += int(s)
counts += 1
while n >= 10:
    new = 0
    while n != 0:
        new += n % 10
        n /= 10
    n = new
    counts += 1
print(counts)
print('YES' if n == 3 or n == 6 or n == 9 else 'NO')